setup.py

```diff
@@ -24,7 +24,7 @@
     author_email='guillaume.camera@people-doc.com, '
                  'guillaume.gerard@people-doc.com',
     install_requires=[
-        'Django<1.10',
+        'Django',
```

Comment:
this silly requirement has almost make me lose my mind ; I was working with tox "django110" jobs, and it was still Django 1.9 that was installed. Luckily I did a careful check to the "pip freeze -l" that is executed during the job, or we could have a "fake" Django 1.10 support.

Maybe we could think about having a "latest" job, that installs the latest Django version, run it through a cron job (I don't know if circle-ci is able to do it, but travis does) and periodically check if the latest django release is compatible, in order to *anticipate* further releases.